### PR TITLE
Created Database Tables for Calling Mod to Thread

### PR DESCRIPTION
**Pull request by akrobi**
Created these three tables: 
- realm_report_webhooks
- user_reports
- user_report_deliveries

### CODE REVIEW

File: db/init/090_subscriptions.sql [hunk 1]
```diff
@@ -43,3 +50,27 @@ CREATE TABLE IF NOT EXISTS thread_category_subscriptions
...
+
+CREATE TABLE IF NOT EXISTS user_reports
+(
+    id BIGINT GENERATED ALWAYS AS IDENTITY PRIMARY KEY NOT NULL,
+    reporter_id BIGINT REFERENCES users(id) ON DELETE RESTRICT NOT NULL,
+    reported_at timestamp NOT NULL DEFAULT now(),
+    text TEXT NOT NULL,
+    reported_thread_id BIGINT REFERENCES threads(id) ON DELETE RESTRICT,
+    reported_post_id BIGINT REFERENCES posts(id) ON DELETE RESTRICT,
+    reorted_comment_id BIGINT REFERENCES comments(id) ON DELETE RESTRICT,
```
Comment (essential-randomness) [inline db/init/090_subscriptions.sql:62]: typo: re**P**orted_comment_id

File: package-lock.json
Comment (essential-randomness) [inline package-lock.json:1]: Revert

File: package.json
Comment (essential-randomness) [inline package.json:1]: Revert (I'll make these changes myself so people don't run into them again)

File: yarn.lock
Comment (essential-randomness) [inline yarn.lock:1]: Revert

**Review by essential-randomness — CHANGES_REQUESTED** [review]
Just one typo and some changes to revert.

To check if the database starts correctly, you can run `yarn run start-db` and make sure the DB doesn't error.

**Comment by akrobi** [comment]
re-submitted under new pull request https://github.com/BobaBoard/boba-backend/pull/140/

**Pull request closed without merging**